### ISSUE TITLE
[Main menu - Site Admin] account menu item role should be button and collapse attribute properly populated

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#6410]
## Area or Component

Main navigation menu with a site admin role.

## Quick Summary

The account button, which upon clicking displays a submenu, has a role of link and no collapsible aria attribute.

### How to Replicate

Check the DOM.

## Accessibility Violation

[The element's role is missing or is not appropriate for the element's function.
](https://docs.deque.com/issue-help/1.0.0/en/aria-role-missing)

[The element has missing or incorrect states or properties that are necessary for screen reader users to interact with or understand the content conveyed by the element.](https://docs.deque.com/issue-help/1.0.0/en/aria-state-property-missing)

## Action Requested

Set the role to button
Add aria-collapse = true/false depending the state of the submenu.

## Screenshots (optional)
<img width="1197" alt="Screenshot 2023-08-24 at 10 29 28 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/ab462f45-c69c-41ef-a173-1c8b5df53e36">

**Comment by johanna-skylight** [comment]
@mehansen the settings button is also displaying even though it does nothing for this role. Let me know if you want me to add the removal of that button as part of the a/c of the ticket.

**Comment by mehansen** [comment]
> @mehansen the settings button is also displaying even though it does nothing for this role. Let me know if you want me to add the removal of that button as part of the a/c of the ticket.

thanks for the callout @johanna-skylight I'm leaning towards leaving this alone but let's bring it up when we refine this ticket. I think it might make sense to at least make design aware so they can consider that in any future header designs

**Comment by mehansen** [comment]
@ whoever takes this ticket: please reach out to design and let them know, but it's not an accessibility violation and we don't have any bugs around this, so no need to hold up resolving this ticket over this. if necessary we can make a followup design fix ticket

**Comment by emyl3** [comment]
https://github.com/CDCgov/prime-simplereport/issues/6410#issuecomment-1697869471
this part is done :) https://skylight-hq.slack.com/archives/C039689GS4S/p1694012183896279
